pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1169,7 +1169,13 @@ func propagatePipelineNameLabelToPipelineRun(pr *v1.PipelineRun) error {
 	case pr.Spec.PipelineSpec != nil:
 		pr.ObjectMeta.Labels[pipeline.PipelineLabelKey] = pr.Name
 	case pr.Spec.PipelineRef != nil && pr.Spec.PipelineRef.Resolver != "":
-		pr.ObjectMeta.Labels[pipeline.PipelineLabelKey] = pr.Name
+		// https://tekton.dev/docs/pipelines/cluster-resolver/#pipeline-resolution
```

Comment:
This code caters to every resolver. So removing will mean no value for `git-resolver`, etc.

Reply:
Sorry, didn't consider Git's resolver.  If it is universal, then this PR should not be necessary.

Reply:
Should we even `PipelineName` label for `resolvers`?

Reply:
According to the documentation, resolver is a pod that parses some data according to the configuration. So, I don't know much about adding the PipelineName tag to the resolver.

For other resolvers, is it possible that they are not associated with a pipeline, so is it reasonable to not have this label?

Reply:
Or we can determine if the kind field is a pipeline, if not, still use the default name of PipelineRun, otherwise, we can choose the correct name of the pipeline.

```yaml
spec:
  pipelineRef:
    params:
    - name: kind
      value: pipeline
```

Reply:
Yes, but it would seem like depending upon the resolver's param in the pipeline controllers code. So I am fine with not having this also.

Reply:
@vdemeester What do you think? Do we need this or not?

Reply:
I don't think we need this. cc @lbernick @abayer

Reply:
Hmmm, I agree putting a pipelinerun name to `tekton.dev/pipeline` is a bit confusing. Today, this logic happens before remote resolution. I'm thinking if we can propagate this value after resolving the pipeline when using a resolver.